ds3-autogen-c/src/main/resources/templates/request-templates/SpectraS3GetRequest.ftl

```diff
@@ -0,0 +1,29 @@
+<#-- ********************************** -->
+<#-- Generate "SpectraS3 Get Request" -->
+<#--   Input: Source object             -->
+<#-- ********************************** -->
+<#list getRequests() as requestEntry>
+    <#if (requestEntry.getClassification().toString() == "spectrads3") && (requestEntry.getVerb().toString() == "GET")>
```

Comment:
It seems like this should already be filtered out in the generator before getting to code gen.